### PR TITLE
Add menuconfig option to invert colors on ILI9341

### PR DESCRIPTION
**Pull request by rangeli**
This fixes issue #119

### CODE REVIEW

**Comment by rangeli** [comment]
@C47D

File: components/lvgl_esp32_drivers/lvgl_tft/ili9341.h [hunk 1]
```diff
@@ -40,6 +40,7 @@ extern "C" {
 #endif
 
 #define ILI9341_INVERT_DISPLAY CONFIG_LVGL_INVERT_DISPLAY
+#define ILI9341_INVERT_COLORS LVGL_INVERT_COLORS
```
Comment (C47D) [inline components/lvgl_esp32_drivers/lvgl_tft/ili9341.h]: `LVGL_INVERT_COLORS` must be `CONFIG_LVGL_INVERT_COLORS`, the `CONFIG` prefix is added.

File: components/lvgl_esp32_drivers/lvgl_tft/ili9341.c [hunk 2]
```diff
@@ -159,6 +159,12 @@ void ili9341_init(void)
 	ili9341_send_cmd(0x36);
 	ili9341_send_data(&data, 1);
 #endif
+
+#if defined ILI9341_INVERT_COLORS
```
Comment (C47D) [inline components/lvgl_esp32_drivers/lvgl_tft/ili9341.c]: Check `build/config/sdkconfig.h` to know what value `CONFIG_LVGL_INVERT_COLORS` have if you set to option to false, I don't know if the option is set to 0 or simply not defined.

If it is set to 0 when you don't enable the option then I think this must be:
```c
#if ILI9341_INVERT_COLORS == 1
```

File: components/lvgl_esp32_drivers/lvgl_tft/ili9341.h (hunk 1 above)
Comment (mica-angeli) [reply components/lvgl_esp32_drivers/lvgl_tft/ili9341.h]: Fixed

File: components/lvgl_esp32_drivers/lvgl_tft/ili9341.c (hunk 2 above)
Comment (mica-angeli) [reply components/lvgl_esp32_drivers/lvgl_tft/ili9341.c]: The `CONFIG_LVGL_INVERT_COLORS` value equals `1` when it's set and it's undefined when not set.

I changed it to check the value and it's working correctly both when the color inversion option is set and when it is not set.

**Comment by C47D** [comment]
Thanks for the PR, I can't test it right now, so will test it later today.

**Pull request merged**